### PR TITLE
Update URLconfs to use django.urls.path()

### PR DESCRIPTION
**Pull request by cansarigol**

### CODE REVIEW

File: cab/urls/bookmarks.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-        bookmarks.user_bookmarks,
-        name='cab_user_bookmarks'),
-    url(r'^add/(?P<snippet_id>\d+)/$',
-        bookmarks.add_bookmark,
-        name='cab_bookmark_add'),
-    url(r'^delete/(?P<snippet_id>\d+)/$',
-        bookmarks.delete_bookmark,
-        name='cab_bookmark_delete'),
+    path('',
+         bookmarks.user_bookmarks,
```
Comment (timgraham) [inline cab/urls/bookmarks.py]: I'd like to follow Django's guideline of allowing lines long than 79 characters if it helps readability.  I would single line all the path()s

File: ratings/converters.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class FloatConverter:
+    regex = '[+-]?([0-9]*[.])?[0-9]+'
```
Comment (timgraham) [inline ratings/converters.py]: Is the plus sign required? I don't see it in the orignal regex or any test failing without it.

File: ratings/converters.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+    def to_python(self, value):
+        return value
```
Comment (timgraham) [inline ratings/converters.py]: You can move the some type conversion from the `rate_object` view to here.

File: cab/urls/bookmarks.py (hunk 1 above)
Comment (cansarigol) [reply cab/urls/bookmarks.py]: I agree with you but I didn't want to change these lines. I changed now

File: ratings/converters.py (hunk 2 above)
Comment (cansarigol) [reply ratings/converters.py]: you are right, I erased it

**Pull request merged**